influx_on_ubuntu/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Influx on Ubuntu",
```

Comment:
Would be good to add blockchain in here as well so the name is clearer in the Azure.com index